src/export/exportChannelColumn.component.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Button,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+} from '@mui/material';
+import type { AxiosError } from 'axios';
+import React from 'react';
+import { useExportData } from '../api/export';
+import type { FullChannelMetadata } from '../app.types';
+import handleOG_APIError from '../handleOG_APIError';
+
+export interface ExportChannelColumnProps {
+  open: boolean;
+  onClose: () => void;
+  channelInfo: FullChannelMetadata;
+}
+
+const ExportChannelColumn = (props: ExportChannelColumnProps) => {
+  const { open, onClose, channelInfo } = props;
+
+  const { mutateAsync: exportChannels, isPending } = useExportData();
+
+  const handleExportChannel = React.useCallback(() => {
+    exportChannels({
+      exportType: 'All Rows',
+      dataToExport: {
+        Scalars: true,
+        Images: true,
+        'Waveform CSVs': true,
+        'Waveform Images': true,
```

Comment:
Might need to check if we need this - I'd go with it set to false for now as that is the behaviour of `eCat`

Reply:
is that just for waveform Images ?

Reply:
If so, I'll also set scalar to false, as this dialog only exports image and waveform channels.

Reply:
Yeah, just for waveform thumbnails as in `eCat` just exports the CSVs